docs/Documentation.md

```diff
@@ -1,54 +1,63 @@
-  * [inContext](#incontext)
-  * [Reusing Elements](#reusing-elements)
-  * [Missing Keys](#missing-keys)
-  * [Modifiers FAQ](#modifiers-faq)
-* [Reactive Data](#reactive-data)
-  * [Attributes and Properties](#attributes-and-properties)
-  * [Event Streams and Signals](#event-streams-and-signals)
-  * [Individual Children](#individual-children)
-  * [Lists of Children](#lists-of-children)
-    * [`children <-- observableOfElements`](#children----observableofelements)
-    * [Performant Children Rendering – `split`](#performant-children-rendering--split) 
-    * [Performant Children Rendering – `children.command`](#performant-children-rendering--childrencommand)
-  * [Binding Observables](#binding-observables)
-  * [Other Binders](#other-binders)
-* [ClassName and Other Special Keys](#classname-and-other-special-keys)
-  * [cls](#cls)
-  * [Other Composite Keys](#other-composite-keys)
-  * [SVG](#svg)
-* [Event System: Emitters, Transformations, Buses](#event-system-emitters-transformations-buses)
-  * [Registering a DOM Event Listener](#registering-a-dom-event-listener)
-  * [EventBus](#eventbus)
-  * [Alternative Event Listener Registration Syntax](#alternative-event-listener-registration-syntax)
-  * [Multiple Event Listeners](#multiple-event-listeners)
-  * [Event Transformations](#event-transformations)
-    * [preventDefault & stopPropagation](#preventdefault--stoppropagation)
-    * [useCapture](#usecapture)
-    * [Obtaining Typed Event Target](#obtaining-typed-event-target)
-  * [Window & Document Events](#window--document-events)
-* [Ownership](#ownership)
-  * [Laminar's Use of Airstream Ownership](#laminars-use-of-airstream-ownership)
-* [Memory Management](#memory-management)
-* [Element Lifecycle Hooks](#element-lifecycle-hooks)
-  * [What Is Mounting?](#what-is-mounting)
-  * [`onMountCallback`](#onmountcallback)
-  * [`onUnmountCallback` & `onMountUnmountCallback`](#onunmountcallback--onmountunmountcallback)
-  * [`onMountSet`](#onmountset)
-  * [`onMountBind`](#onmountbind)
-  * [`onMountInsert`](#onmountinsert)
-  * [Why Use Lifecycle Hooks?](#why-use-lifecycle-hooks)
-  * [Lifecycle Event Timing](#lifecycle-event-timing)
-  * [How Are Mount Events Propagated?](#how-are-mount-events-propagated)
-* [URL Routing](#url-routing)
-* [Special Cases](#special-cases)
+- [Introduction](#introduction)
+    - [First, see Why Laminar and [Quick Start](../README.md#quick-start).](#first-see-why-laminar-and-quick-start)
+    - [Documentation](#documentation)
+- [Imports](#imports)
+- [Tags & Elements](#tags--elements)
+- [Modifiers](#modifiers)
+  - [Nesting and Children](#nesting-and-children)
+  - [Empty Nodes](#empty-nodes)
+  - [Manual Application](#manual-application)
+  - [`inContext`](#incontext)
+  - [Reusing Elements](#reusing-elements)
+  - [Missing Keys](#missing-keys)
+  - [Modifiers FAQ](#modifiers-faq)
+- [Rendering](#rendering)
+- [Reactive Data](#reactive-data)
+  - [Attributes and Properties](#attributes-and-properties)
```

Comment:
I think the Markdown formatting changes should go, in case you really want them, to a separate PR.

Reply:
o_O

I actually didn't do this, it must be something in my markdown VS code setup.

All I wanted was just to remove the header, I'll revert.